rendering/fbx.ksy

```diff
@@ -0,0 +1,148 @@
+meta:
+  id: fbx
+  title: FBX 3D model interchange format
+  file-extension: fbx
```

Comment:
It'd be nice to add `application: Autodesk/Kaydara FiLMBOX`

Reply:
Will add. I had wondered about this. The native application was Filmbox but now it's used by damn near everybody. Everybody using the Autodesk FBX SDK for a start, but then even implementations in open source like Blender's.